### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marin-bratanov**
testing pull requests and comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@ docs-test
 =========
 
 Test repo for docs editing workflow
+
```
Comment (marin-bratanov) [inline README.md]: some comment on a new line

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@ docs-test
 =========
 
 Test repo for docs editing workflow
```
Comment (marin-bratanov) [inline README.md]: comment on existing unchanged line

File: README.md (hunk 1 above)
Comment (marin-bratanov) [inline README.md]: more lines in comments

File: README.md [hunk 3]
```diff
@@ -2,3 +2,8 @@ docs-test
...
+
+
+added lines
```
Comment (marin-bratanov) [inline README.md]: a comment

Comment (marin-bratanov) [inline README.md]: another comment on this line

File: README.md [hunk 4]
```diff
@@ -2,3 +2,8 @@ docs-test
 =========
 
 Test repo for docs editing workflow
```
Comment (marin-bratanov) [inline README.md]: a comment on unchanged line

Comment (Iankodj) [inline README.md]: my comment

**Pull request merged**